.env

```diff
@@ -0,0 +1,9 @@
+# Flask server configuration.
```

Comment:
We don't want to check this file in - this should be used for storing secrets, so it either will leak those to the source code, or it will not have secrets and be useless as is. Instead, in our pipeline we can pass through appropriate environment variables directly, e.g. with `docker run -e MY_ENV <image_tag>`